pkg/storage/SQL/model_plan/get_by_operational_solution_key.sql

```diff
@@ -0,0 +1,39 @@
+SELECT
```

Comment:
I have wondered if this is better implemented as a DataLoader implementation. The cost/benefit proposition on what will typically be small result sets seems dubious, but I am interested in hearing other perspectives on this. In the meantime, I built this out as a quick and simple(r) solution which can be refactored if it makes sense to do so.

Reply:
Thank you for elaborating on your consideration for DataLoader usage. Based on that criteria, I agree 💯.